components/button/button-toggle.js

```diff
@@ -50,6 +55,15 @@ class ButtonToggle extends LitElement {
 		`;
 	}
 
+	updated(changedProperties) {
+		super.updated(changedProperties);
+
+		if (changedProperties.get('pressed') === undefined) return;
+
+		/** Dispatched when the pressed state changes */
+		this.dispatchEvent(new CustomEvent('d2l-button-toggle-change'));
```

Comment:
For `change` events, we have just used `'change'` as the event name for our form inputs since I think we consider it a more built-in type of event. Could use that here -- I'm good either way.

Reply:
Yeah that could certainly work in this case as well.  I'll mull it over while I wait for Jeff's input on the docs. Or if anyone else has a preference I am happy to oblige. I know we have a mix across our components, and in some of those cases there may be reasons to be specific.

Reply:
I did a little poking around and it seems like the `change` event was originally intended for form elements `INPUT`, `SELECT`, and `TEXTAREA`. Unless there is any strong opinion, I'll leave this as `d2l-button-toggle-change` (or just `d2l-button-toggle`).